packages/nextjs/components/scaffold-eth/Contract/ContractUI.tsx

```diff
@@ -54,66 +55,76 @@ const ContractUI = ({ contractName }: TContractUIProps) => {
     [contract, displayedContractFunctions],
   );
 
+  if (deployedContractLoading) {
+    return (
+      <div className="mt-14">
+        <Spinner width="50px" height="50px" />
+      </div>
+    );
+  }
+
   if (!contractAddress) {
-    return <p className="text-2xl">No Contract found!</p>;
+    return <p className="text-3xl mt-14">No Contract found!</p>;
```

Comment:
At some point, we'll have more info here. Like some hints why this might showing.